tests/filecheck/backend/rvscf_lowering_labels.mlir

```diff
@@ -0,0 +1,112 @@
+// RUN: xdsl-opt -p lower-riscv-scf-to-labels --split-input-file %s | filecheck %s
+
+builtin.module {
+    "riscv_func.func"() ({
+    ^0(%0 : !riscv.reg<a0>, %1 : !riscv.reg<a1>):
+        %2 = riscv.mv %0 : (!riscv.reg<a0>) -> !riscv.reg<a0>
+        %3 = riscv.mv %1 : (!riscv.reg<a1>) -> !riscv.reg<a1>
+        %4 = riscv.li 1 : () -> !riscv.reg<a2>
+        %5 = riscv.li 0 : () -> !riscv.reg<a3>
+        %6 = riscv.fcvt.s.w %5 : (!riscv.reg<a3>) -> !riscv.freg<fa0>
```

Comment:
Can we make this an integer loop? That would allow removing the fcvt instructions and would result in a more straightforward loop body. Also, given that floating-point is an extension over the base instruction set, maybe we should avoid using it in the base test cases for features that can work without floating point support

Reply:
I'll add another test with an integer loop in another PR, I think it would be good to keep both.